src/io/gml.c

```diff
@@ -762,6 +902,7 @@ static igraph_error_t igraph_i_gml_convert_to_key(const char *orig, char **key)
  * \param id Either <code>NULL</code> or a numeric vector with the vertex IDs.
  *        See details above.
  * \param creator An optional string to write to the stream in the creator line.
+ *        It must not contain the <code>"</code> character.
```

Comment:
Why not? Aren't we encoding it now as `&quot;`?

Reply:
Because I was lazy. I can fix this, no problem.